apps/api/src/app/domains/userProfiles/commands/create-user-profile.command.ts

```diff
@@ -28,6 +28,10 @@ export class CreateUserProfileHandler
 
     const aggregate = new UserProfileAggregate();
 
+    if (data.socialLinks[0] == null) {
```

Comment:
if you want to check or array got null you should check like this: data.socialLinks.includes(null)